### ISSUE TITLE
Value Set Expansion Error (Wrong Code?)  [Medium-High Priority]

### ISSUE THREAD

**Issue by markkramerus** [standardhealth/shr-fhir-export#38]
Some of our value sets are not getting into the Implementation Guide correctly. For example, SHR LymphSystemSubdivisionVS ValueSet:

**Expansion
Error expanding ValueSet "null: Error unmarshalling resource: Unable to provide support for code system http://snomed.info/sct version http://snomed.info/sct/731000124108**

The definition of LymphSystemSubdivisionVS doesn't use this code, however:

ValueSet:	LymphSystemSubdivisionVS
Includes codes descending from SCT#123849002 "Lymphoid system subdivision (body structure)"

I'm not sure if this is a code system versioning problem a terminology server problem, a problem in how we are describing implicit value sets, or an error in how we are accessing the terminology server API.

**Comment by cmoesel** [comment]
I'm not sure where that `http://snomed.info/sct/731000124108` reference is coming from.  If we look at the JSON representation of the value set, we specify the system only as `"http://snomed.info/sct"` and are properly indicating composition via `is-a` `123849002`.

It seems like it may be a terminology server issue, but I'll dig a little deeper to see if I can find where that weird version URI is popping up and to confirm that `123849002` is still a valid code in SNOMED (although I don't see why it wouldn't be).

Here's the JSON representation, BTW:

```json
{
  "resourceType": "ValueSet",
  "id": "shr-oncology-LymphSystemSubdivisionVS",
  "meta": {
    "profile": [
      "http://hl7.org/fhir/StructureDefinition/valueset-shareable-definition"
    ]
  },
  "text": {
    "status": "generated",
    "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">\n<p><b>SHR LymphSystemSubdivisionVS ValueSet</b></p>\n<p/>\n</div>"
  },
  "url": "http://standardhealthrecord.org/shr/oncology/vs/LymphSystemSubdivisionVS",
  "identifier": [
    {
      "system": "http://standardhealthrecord.org",
      "value": "shr.oncology.LymphSystemSubdivisionVS"
    }
  ],
  "version": "0.0.1",
  "name": "SHR LymphSystemSubdivisionVS ValueSet",
  "title": "SHR LymphSystemSubdivisionVS ValueSet",
  "status": "draft",
  "date": "2017-07-31T17:47:51.169Z",
  "publisher": "The MITRE Corporation: Standard Health Record Collaborative",
  "contact": [
    {
      "telecom": [
        {
          "system": "url",
          "value": "http://www.standardhealthrecord.org"
        }
      ]
    }
  ],
  "compose": {
    "include": [
      {
        "system": "http://snomed.info/sct",
        "filter": [
          {
            "property": "concept",
            "op": "is-a",
            "value": "123849002"
          }
        ]
      }
    ]
  }
}
```

**Comment by cmoesel** [comment]
Found it.  We do call out that specific version of SNOMED-CT in our shr.json IG config file.  I believe we pulled that direct from the FHIR documentation, but it sounds like maybe the terminology server dropped support for it.  I'll try to determine what version we _should_ be using and move to that.

**Comment by cmoesel** [comment]
After some research, this _is_ the valid value to indicate you want the _U.S._ version of SNOMED-CT.  See the FHIR IG publisher documentation here:
http://wiki.hl7.org/index.php?title=IG_Publisher_Documentation#Terminology_Options

So... it appears that perhaps the terminology server is no longer supporting the U.S. edition of SNOMED-CT?  More investigation is needed.

**Comment by cmoesel** [comment]
OK.  I was able to fix this by updating our definitions and IG publisher jar from the 3.0.0 versions to the 3.0.1 version.  The FHIR team must have fixed something in there.  Will ensure it gets into the next revision of the FHIR exporter.

**Comment by cmoesel** [comment]
Fixed in shr-fhir-export v5.0.0-beta.4 and shr-cli v5.0.0-beta.5.
